packages/types/src/types/cloudinary-video-player.ts

```diff
@@ -27,11 +28,11 @@ export interface CloudinaryVideoPlayerOptions extends CloudinaryVideoPlayerOptio
   fluid?: boolean;
   fontFace?: string;
   hideContextMenu?: boolean;
-  interactionAreas?: any; // Used "any" cause this fetaure is still in beta
-  playbackRates?: Array<any>;
+  interactionAreas?: unknown; // Used "any" cause this fetaure is still in beta
```

Comment:
nitpick: I think you can remove the comment here because it is no longer up to date

Reply:
Good catch! Fixed